entity-view/api/src/main/java/com/blazebit/persistence/view/metamodel/SourceType.java

```diff
@@ -0,0 +1,18 @@
+package com.blazebit.persistence.view.metamodel;
+
+public interface SourceType<X> extends Type<X> {
```

Comment:
@beikov I'm not sure if it is a good idea for me to split these interfaces, but in my mind it made sense to have a separate interface for sources and targets when converting?